mu-plugins/blocks/favorite-button/src/block.json

```
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "wporg/favorite-button",
+	"version": "0.1.0",
+	"title": "Favorite Button",
+	"category": "design",
+	"icon": "",
+	"description": "A button to toggle favoriting on the current item.",
+	"textdomain": "wporg",
+	"supports": {
+		"html": false
+	},
+	"attributes": {
+		"showCount": {
+			"type": "boolean",
+			"default": false
+		},
+		"variant": {
+			"type": "string",
+			"enum": [ "default", "small" ],
```

For me the variants are more obvious as 
```suggestion
			"enum": [ "default", "no-outline" ],
```

---

It does also use a smaller padding, and is designed for "small" spaces, so I would still prefer `small`. Along the lines of the `is-small` [helper class for buttons](https://github.com/WordPress/wporg-parent-2021/blob/c251ec4e09de44fa3fd68c9ba95bb0bca220903e/source/wp-content/themes/wporg-parent-2021/sass/blocks/_button.scss#L15).